genai-perf/genai_perf/telemetry_data/telemetry_data_collector.py

```diff
@@ -46,6 +46,16 @@ def __init__(
         self._stop_event = Event()
         self._thread: Optional[Thread] = None
 
+    def check_url_reachability(self) -> bool:
+        """Check if the server metrics URL is reachable"""
+        if self._server_metrics_url:
+            try:
+                response = requests.get(self._server_metrics_url, timeout=5)
```

Comment:
Set the timeout value to a separate variable that clarifies the units. We want to avoid magic numbers.